### PR TITLE
Release config: update subchart app versions via renovate, comment behavior

### PR DESCRIPTION
**Pull request by pat-s**

### CODE REVIEW

File: release-config.ts [hunk 1]
```diff
@@ -1,19 +1,15 @@
 export default {
   beforePrepare: async ({ exec, nextVersion }) => {
     await exec(`apk add --no-cache helm curl`);
+    // This updates the 'version' in Chart.yaml, inferred from the version which is supposed to be released
     await exec(`sed -i "s/^version:.*$/version: ${nextVersion}/g" charts/woodpecker/Chart.yaml`);
+    // This updates the appVersion in Chart.yaml with the latest Woodpecker version from woodpecker-ci/woodpecker
     await exec(
-      `sed -i '' 's/^version:/version: ${nextVersion}/' charts/woodpecker/charts/agent/Chart.yaml`
+      `export UPSTREAM_VERSION=$(curl -s https://api.github.com/repos/woodpecker-ci/woodpecker/releases/latest | grep tag_name | grep -Eo "([0-9]+(\.[0-9]+)+)") && sed -i "s/^appVersion:.*$/appVersion: $UPSTREAM_VERSION/g" charts/woodpecker/Chart.yaml charts/woodpecker/charts/agent/Chart.yaml charts/woodpecker/charts/server/Chart.yaml`,
```
Comment (anbraten) [inline release-config.ts:8]: No sure if we should automatically update the app version as a bump would be missing in the changelog and might be unintended by the chart user.

Comment (pat-s) [reply release-config.ts:8]: We wouldn't do a standalone PR just for the appVersion, would we? So the bump would most likely be added manually anyhow.

> might be unintended by the chart user.

What do you mean by this? I don't understand it.

The reasoning here is: this only happens on a release and when we do a release, I can't see a case in which we don't want to be on the latest appVersion (?). And if we are and WP hasn't changed, this command won't do anything.

Comment (anbraten) [reply release-config.ts:8]: Issue would be that you can only release commits, so if we only want to release a bump of the app-version there would be no commit in this setup. Instead I had in mind that renovate creates a PR bumping the app version and then a new release is created using the release-helper.

Comment (pat-s) [reply release-config.ts]: Ah good idea! I've converted the management of `appVersion` for all charst to renovate, see recent commits.

File: release-config.ts [hunk 2]
```diff
@@ -1,19 +1,11 @@
 export default {
   beforePrepare: async ({ exec, nextVersion }) => {
     await exec(`apk add --no-cache helm curl`);
+    // This updates the 'version' in Chart.yaml, inferred from the version which is supposed to be released
```
Comment (anbraten) [inline release-config.ts:4]:
```suggestion
```

Just redundant to the command.

**Review by anbraten — APPROVED** [review]

**Pull request merged**